marlowe-playground-client/src/SaveAs/Types.purs

```diff
@@ -0,0 +1,29 @@
+module SaveAs.Types where
```

Comment:
This looks like a straight duplicate of `marlowe-playground-client/src/Rename/Types.purs`?

Reply:
interesting, indeed they are exactly the same and I could get rid of one and it should all work fine however what would I call it and is it worth it, the separation is nice in a way.